packages/react/src/Alert/Alert.tsx

```diff
@@ -3,22 +3,33 @@
  * Copyright (c) 2023 Gemeente Amsterdam
  */
 
-import { AlertIcon, CheckmarkIcon, CloseIcon } from '@amsterdam/design-system-react-icons'
+import { AlertIcon, CheckmarkIcon, CloseIcon, InfoIcon } from '@amsterdam/design-system-react-icons'
 import clsx from 'clsx'
-import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren, useMemo } from 'react'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { Heading } from '../Heading'
+import type { HeadingProps } from '../Heading'
 import { Icon } from '../Icon'
 import { VisuallyHidden } from '../VisuallyHidden'
 
 export interface AlertProps extends PropsWithChildren<HTMLAttributes<HTMLDivElement>> {
-  title?: string
-  severity?: undefined | 'error' | 'success'
+  /** Whether the alert can be dismissed by the user. Adds a button to the top right. */
   closeable?: boolean
-  icon?: boolean
+  /**
+   * The hierarchical level of the alert title within the document.
+   * @default 4
```

Comment:
```suggestion
   * @default 2
```

Reply:
Sharp as a knife, you :)

I’ll add a task to decide explicitly how far we want to go with JSDoc comments.